nerfstudio/models/nerfacto.py

```diff
@@ -283,7 +285,14 @@ def get_outputs(self, ray_bundle: RayBundle):
         weights_list.append(weights)
         ray_samples_list.append(ray_samples)
 
-        rgb = self.renderer_rgb(rgb=field_outputs[FieldHeadNames.RGB], weights=weights)
+        background_color = (
+            self.renderer_rgb.get_background_color(rgb=field_outputs[FieldHeadNames.RGB], background_color="random")
```

Comment:
background_color="random" is not correct here, is it? Isn't it part of the config?

Reply:
I agree if we decide to remove the `use_alpha_transparency_training` argument. Otherwise, it would make more sense to keep it hard coded.